docker/api/cnaas-setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+set -x
+
+export DEBIAN_FRONTEND noninteractive
+
+/bin/sed -i s/deb.debian.org/ftp.se.debian.org/g /etc/apt/sources.list
+
+apt-get update && \
+    apt-get -y dist-upgrade && \
+    apt-get install -y \
+      git \
+      python3-venv \
+      python3-pip \
+      python3-yaml \
+      iputils-ping \
+      procps \
+      bind9-host \
+      netcat-openbsd \
+      net-tools \
+      curl \
+      netcat \
+      nginx \
+      supervisor \
+    && apt-get clean
+
+pip3 install uwsgi
+
+# Start venv
+python3 -m venv /opt/cnaas/venv
+cd /opt/cnaas/venv/
+source bin/activate
+
+/opt/cnaas/venv/bin/pip install -U pip
+
+# Fetch the code and install dependencies
+git clone https://github.com/SUNET/cnaas-nms.git
```

Comment:
Is this really a good idea? I'd like to be sure that tagged containers really runs a specific version of the code.

Reply:
I think this is the same as for other containers, cnaas-setup.sh is only run once at build time of the container. But I changed so we don't git pull on runtime (exec-pre-app.sh), that was only for development purposes